src/main/java/frc/robot/commands/KvCharacterization.java

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.math.kinematics.ChassisSpeeds;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.PoseEstimation;
+import frc.robot.subsystems.SwerveSubsystem;
+import frc.robot.swerve.SwerveModuleAbstract;
+
+public class KvCharacterization extends Command {
+  /** Creates a new kV_Characterization. */
+  private final SwerveSubsystem mSwerve;
+  private final SwerveModuleAbstract[] mModules;
+  private final Timer mTimer = new Timer();
+  private final double mTargVelMPS;
+  private double increment = 0;
+
+  private double [] expkV = new double[4];
```

Comment:
Naming convention...

Reply:
And avoid using "4" as is here and below, use a constant with a mnemonic name instead.

Reply:
done